java-core/src/test/java/ai/certifai/junit5/ex2/GetSetMethodAssertionTestSolution.java

```diff
@@ -0,0 +1,43 @@
+package ai.certifai.junit5.ex2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GetSetMethodAssertionTestSolution {
+    @Test
+    void getCoffeeType() {
```

Comment:
I suggest you to follow the convention on how people name their test case. you can refer to this link for more information: https://dzone.com/articles/7-popular-unit-test-naming